README.md

```diff
@@ -1,50 +1,1323 @@
+
+   ```
+   $ cp ~/ODIM/odim-controller/helmcharts/aciplugin/aciplugin-config.yaml ~/plugins/aciplugin
+   ```
+
+5. Open the Cisco ACI plugin configuration YAML file.
+
+   ```
+   $ vi ~/plugins/aciplugin/aciplugin-config.yaml
+   ```
+
+   **Sample aciplugin-config.yaml file:**
+
+   ```
+   odimra:
+    namespace: odim
+    groupID: 2021
+   aciplugin:
+    hostname: knode1
+    eventListenerNodePort: 30084
+    aciPluginRootServiceUUID: 7a38b735-8b9f-48a0-b3e7-e5a180567d37
+    username: admin
+    password: sTfTyTZFvNj5zU5Tt0TfyDYU-ye3_ZqTMnMIj-LAeXaa8vCnBqq8Ga7zV6ZdfqQCdSAzmaO5AJxccD99UHLVlQ==
+    lbHost: 10.24.1.232
+    lbPort: 30084
+    logPath: /var/log/aciplugin_logs
+    haDeploymentEnabled: false
+   aciplugin:
+    eventListenerNodePort: 30086
+    aciPluginRootServiceUUID: a127eedc-c29b-416c-8c82-413153a3c351
+    username: admin
+    password: sTfTyTZFvNj5zU5Tt0TfyDYU-ye3_ZqTMnMIj-
+    LAeXaa8vCnBqq8Ga7zV6ZdfqQCdSAzmaO5AJxccD99UHLVlQ==
+    lbHost: 10.24.1.232
+    lbPort: 30086
+    logPath: /var/log/aciplugin_logs
+    apicHost: 10.42.0.50
+    apicUserName: admin
+    apicPassword: Apic123
+    odimURL: https://api:45000
+    odimUserName: admin
+    odimPassword: Od!m12$4
+   ```
+8. Update the following mandatory parameters in the plugin configuration file:
+
+   - **hostname**: Hostname of the cluster node where the Cisco ACI plugin will be installed.
+   - **lbHost**: IP address of the cluster node where the Cisco ACI plugin will be installed.
+   - **lbPort**: Default port is 30084.
+   - **aciPluginRootServiceUUID**: RootServiceUUID to be used by the Cisco ACI plugin service.
+   
+   Other parameters can either be empty or have default values. Optionally, you can update them with values based on your requirements. For more information on each parameter, see [Plugin configuration parameters](#plugin-configuration-parameters).
+   
+9. Generate the Helm package for the Cisco ACI plugin on the deployment node:
+
+   1. Navigate to `odim-controller/helmcharts/aciplugin`.
+
+      ```
+      $ cd ~/ODIM/odim-controller/helmcharts/aciplugin
+      ```
+
+   2. Run the following command to create `aciplugin` Helm package at `~/plugins/aciplugin`:
+
+      ```
+      $ helm package aciplugin -d ~/plugins/aciplugin
+      ```
+
+      The Helm package for the Cisco ACI plugin is created in the tgz format.
+
+8. Save the Cisco ACI plugin Docker image on the deployment node at `~/plugins/aciplugin`.
+
+   ```
+   $ sudo docker save aciplugin:1.0 -o ~/plugins/aciplugin/aciplugin.tar
+   ```
+9. If it is a three-node cluster configuration, log in to each cluster node and [configure proxy server for the plugin](#configuring-proxy-server-for-a-plugin-version). 
+
+   Skip this step if it is a one-node cluster configuration.
+
+10. Navigate to the `/ODIM/odim-controller/scripts` directory on the deployment node.
+
+    ```
+    $ cd ~/ODIM/odim-controller/scripts
```

Comment:
8th step in the pdf file needs to be run here "Update the following parameters in the kube_deploy_nodes.yaml file to their corresponding values:"